### PR TITLE
created manual webview element to allowpopups

### PR DESCRIPTION
**Pull request by jeanawhou**
ticket: https://app.ora.pm/p/320746?c=5381531
WDYT @cameronsutter @Quiescent ?

preview
https://user-images.githubusercontent.com/19387007/124499052-f444bd80-ddef-11eb-8475-e679c08ad077.mp4

### CODE REVIEW

File: src/dashboard/components/help/HelpHome.js [hunk 1]
```diff
@@ -24,6 +24,13 @@ export default function HelpHome(props) {
     }
   }
 
+  const RenderWebView = () => {
+    return React.createElement('webview', {
```
Comment (Quiescent) [inline src/dashboard/components/help/HelpHome.js:28]: Isn't this the same as `<webview src="https://docs.plottr.com" allowpopups></webview>`?
(Where the critical change is that we allowed popups.)

Comment (Quiescent) [reply src/dashboard/components/help/HelpHome.js:28]: If so, then I'd suggest we inline it, rather than creating a separate function for it.

Comment (jeanawhou) [reply src/dashboard/components/help/HelpHome.js:28]: @Quiescent i cant make it work using inline like what's on the docs, i believe the custom prop is being ignored 😕

Comment (Quiescent) [reply src/dashboard/components/help/HelpHome.js:28]: Aha.  Ok.  Pitty :/

**Review by Quiescent — APPROVED** [review]
I've tested that this works and it's fine on my Mac.  LGTM :)

**Pull request merged**